electron/protocol/index.js

```diff
@@ -0,0 +1,32 @@
+const { app, protocol, webContents } = require('electron');
+const log = require('../logger');
+
+const getMainWebContents = () => webContents
+  .getAllWebContents()
+  .find((webcontent) => !!webcontent.browserWindowOptions);
+
+function protocolHandler(event, url) {
+  if (event) { event.preventDefault(); }
+  // need to ignore other windows such as chrome dev tools
+  const wc = getMainWebContents();
+
+  if (!wc) {
+    log.info('not sure we ever actually hit this anymore');
```

Comment:
should remove this -- pretty sure its not used. I havent seen anyt of these logs

Reply:
which means also taking out the timeout